instant/flags.vim

```diff
@@ -111,13 +111,24 @@ call s:plugin.Flag('shfmt_executable', 'shfmt')
 ""
 " Command line arguments to feed prettier. Either a list or callable that
 " takes no args and returns a list with command line arguments.
-call s:plugin.Flag('prettier_options', [
-    \ '--single-quote', '--trailing-comma=all',
-    \ '--arrow-parens=always', '--print-width=80'])
+call s:plugin.Flag('prettier_options', [])
 
 ""
-" The path to the prettier executable.
-call s:plugin.Flag('prettier_executable', 'prettier')
+" @private
+function s:PrettierExecutable() abort
```

Comment:
Consider renaming this to "LookupPrettierExecutable" or something especially self-documenting like that since the name will appear in the output of `:Glaive codefmt`, and it'd take a little effort for the typical user who sees it to track down exactly what it's doing.

Reply:
Done